index.html

```diff
@@ -0,0 +1,386 @@
+
+### Single foldable display
+
+A <dfn>single foldable display</dfn> (SFD) refers to a [=foldable-screen device=] with a single screen.
+
+### Posture
+
+A <dfn>posture</dfn> is defined as the primary way in which the [=foldable device=] is positioned and presented to the user. Because [=foldable devices=] have a hinge they can present different [=postures=] depending on the hinge angle, device orientation, and the way on how the device is being held by the user. Example postures include 'book', 'flat', 'laptop', and 'tent'. The list of postures is expected to evolve over time, and not all foldable devices are able to support all possible postures.
+
+<div class="issue">
+We've discussed the need to rethink the names of the postures to make them as abstract as possible. The fact that 'laptop' is both an actual device and a 'posture' mode can be confusing and misleading. Can a laptop have another posture that's not laptop? Does a mobile device get into laptop posture? Similar case with tablet as a device type and posture.
+</div>
+
+### Window segments
+
+A <dfn>window segment</dfn> represents the regions (and their dimensions) of the window that reside on separate (adjacent) [=displays=].
+
+### Spanning
+
+<dfn>Spanning</dfn> is a window mode on a [=dual-screen device=], where a single application or web content window is spread across both [=screens=] of the device. The ability to maximize a given application across the two [=screens=] is enabled through the [=window manager=]. Because spanning across the two screens double the screen estate it is expected that applications and websites leverage that additional space to optimize or enhance the UX to provide a better experience.
+
+<div class="note">
+A device can be a [=foldable device=] and have a [=multi-monitor setup=]. These [=form factors=] are not mutually exclusive and depend on the hardware present on the device and its use across software.
+</div>
+
+### Window interface
+
+The [=Window interface=] represents a window containing a DOM document, with its document property pointing to the DOM document. The Window interface hosts a number of globally available functions and namespaces, objects, and constructors associated with the concept of a user interface window. In tabbed browsing, each tab is represented by its own Window object.
+
+### Viewport
+
+<div class="issue">
+[Visual Viewport API](#visual-viewport-api) defines the two conceptual viewports, colloquially known as the [=visual viewport=] and [=layout viewport=]. This API may need changes to be compatible with the [related web specifications](#related-web-specifications).
+</div>
+
+### Fold
+
+<dfn>Fold</dfn> is used as both a verb and a noun in the context of this explainer. _To fold_ means to lay one [part of the] screen over another [part of the] screen. _A fold_ (aka a screen fold) as a noun refers to the flexible [=screen area=] that bends.
```

Comment:
Perhaps "refers to the flexible [=screen area=] that bends, or the hinge between the screens of a [=dual-screen device=]", if that's also a fold?

Reply:
Addressed by 623fa72 with some tweaks. @kenchris reviewed and was happy.